src/components/ships/Square.css

```diff
@@ -13,3 +13,27 @@
   cursor: pointer;
   background-color: #004d4d;
 }
+
+.vh {
+  border-color: yellow;
```

Comment:
maybe change this to 
border: 2px solid yellow
then it's thicker ;) and then people can see it. We can also make it green to show it's allowed

Reply:
Or maybe red, put that in the todo list

Reply:
hahaha again XD but I'd say red signalises that it's forbidden. so mayyyybe not red.